### DISCUSSION TITLE
Anyone have any ideas how to fix this?

### DISCUSSION THREAD

**Discussion by unevoljitelj** [Joshua-Riek/ubuntu-rockchip#1087] [General]
https://ibb.co/zfbRkpf

So basicaly, from the first builds of jr ubuntu i had this intermittent issue where monitor goes half to sleep. Image goes dark but backlight remains. Sometimes it goes to sleep properly and sometimes this happens. I thought it was the monitor bcos noone else had thw issue. Well, got a new monitor and voila, same thing. So i have to turn off the monitor.
Any ideas? Any logs i can provide?

Thanks for your time!

**Comment by watou** [comment]
100% of the time for every reboot, after login the screen goes to black on my Rock 5B (not on standby but black), and turning the monitor off and then on causes the desktop to appear correctly. I mention only on the chance there is the same root cause to your issue.

**Reply by unevoljitelj** [reply]
well, this doesnt happen to me but i will try to turn on and off and see what happens
